src/lib.rs

```diff
@@ -70,19 +70,19 @@ extern "C" {
 pub fn height_to_flags(height: u32) -> u32 {
 
     let mut flag = VERIFY_NONE;
-    if height > 170059 {
+    if height >= 173059 {
```

Comment:
From the patch description I don't think this change should be in here. Since it is changed again in the subsequent patch perhaps we could just do the change once there?

Reply:
Updated

Reply:
Face palm, I just spent 2 minutes trying to work out why the first patch did not use `>=` for _all_ the checks - then I remembered it did before I got involved :)